src/Compilers/Core/Portable/Diagnostic/SuppressionDescriptor.cs

```diff
@@ -76,10 +76,9 @@ public bool Equals(SuppressionDescriptor other)
             }
 
             return
-                other != null &&
-                this.Id == other.Id &&
-                this.SuppressedDiagnosticId == other.SuppressedDiagnosticId &&
-                this.Justification.Equals(other.Justification);
+                other is
+            {
+                } && this is { Id: other.Id, SuppressedDiagnosticId: other.SuppressedDiagnosticId } && this.Justification.Equals(other.Justification);
```

Comment:
>other [](start = 35, length = 5)

Not a valid pattern

Reply:
➡️ Treating this one as a duplicate of https://github.com/dotnet/roslyn/pull/39738#discussion_r344405091